source/sdk/threading/native/ConditionWin32.ooc

```diff
@@ -0,0 +1,87 @@
+version(windows) {
+
+	import ../Thread
+	import native/win32/[types, errors]
+	import MutexWin32
+	import structs/ArrayList
+
+	include windows
+
+	CreateEvent: extern func (...) -> Handle
+	SetEvent: extern func (Handle) -> Bool
+	CloseHandle: extern func (Handle) -> Bool
+	INFINITE: extern Long
+	WAIT_OBJECT_0: extern Long
```

Comment:
This is only a cosmetic change, can I do that on separate pull request, or at least when I have access to a windows machine ? I know that this code worked, and if I change something then I will have to ask Jonathan to test it again.